src/ctype/CTypeSchema.ts

```diff
@@ -16,12 +16,11 @@ export const CTypeModel = {
     $schema: {
       type: 'string',
       format: 'uri',
-      default: 'http://kilt-protocol.org/draft-01/ctype#',
-      enum: ['http://kilt-protocol.org/draft-01/ctype#'],
+      const: 'http://kilt-protocol.org/draft-01/ctype#',
     },
     type: {
       type: 'string',
-      enum: ['object'],
+      const: 'object',
```

Comment:
Is this a new JSON schema keyword?

Reply:
It was already available in json draft-07, which we are still using. We just didn't use `const`.

Reply:
Awesome!

Reply:
We will need to change the demo-client accordingly. 

Do we need a ticket?

Reply:
I checked the demo-client and no change is needed.